sos/report/plugins/kdump.py

```diff
@@ -102,4 +103,26 @@ def setup(self):
             "/etc/default/kdump-tools"
         ])
 
+
+class CosKDump(KDump, CosPlugin):
+
+    option_list = [
+        PluginOpt("all_dumps", "enable capture for all kernel dumps",
+                  "", False),
+        PluginOpt("latest_dump", "enable capture for latest kernel crash dump",
+                  "", False),
```

Comment:
`PluginOpt` does not take the same order of parameters as the option tuples. Please re-look at the reference options in other plugins. `PluginOpt("opt-name", default=<default value>, desc="short description")` is what's needed at minimum.